init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/service/JsonService.java

```diff
@@ -0,0 +1,55 @@
+package uk.ac.ox.ndph.mts.init_service.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.ac.ox.ndph.mts.init_service.config.GitRepo;
+import uk.ac.ox.ndph.mts.init_service.exception.InvalidConfigException;
+import uk.ac.ox.ndph.mts.init_service.model.Trial;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+
+@Service
+public class JsonService {
+
+    @Value("${config.json}")
+    private String config;
+
+    private Trial trial;
+    private GitRepo gitRepo;
+
+    @Autowired
+    public JsonService(GitRepo gitRepo) {
+        this.gitRepo = gitRepo;
+    }
+
+    protected Trial createTrialFromGitRepo(String fileName) {
```

Comment:
1. Isn't better to name the model TrialConfig since it probably include other this apart from the actual "trial"?
2. I think we shouldn't say where the file is coming from, so all the "GitRepo" mentions in this service should be remove.

Reply:
Renamed to TrialConfig, 

Placed a TODO to rename GitRepo, we will make these changed once we introduce config-server.